DB/src/main/java/io/deephaven/db/util/PythonScopeJpyImpl.java

```diff
@@ -67,6 +67,72 @@ public CallableWrapper(PyObject pyObject) {
         public Object call(Object... args) {
             return convert(pyObject.callMethod("__call__", args));
         }
+
+        public PyObject getPyObject() {
+            return pyObject;
+        }
+    }
+
+    public static final class NumbaCallableWrapper extends CallableWrapper {
+        private List<Class> paramTypes;
+        private Class returnType;
+
+        public NumbaCallableWrapper(PyObject pyObject, Class returnType, List<Class> paramTypes) {
+            super(pyObject);
+            this.returnType = returnType;
+            this.paramTypes = paramTypes;
+        }
+
+        public Class getReturnType() {
+            return returnType;
+        }
+
+        public List<Class> getParamTypes() {
+            return paramTypes;
+        }
+    }
+
+    private static CallableWrapper wrapCallable(PyObject pyObject) {
+        if (pyObject.hasAttribute("ufunc")) {
+            List<PyObject> params = pyObject.getAttribute("types").asList();
+            if (params.isEmpty()) {
+                return new CallableWrapper(pyObject);
+            }
+            String numbaFuncTypes = params.get(0).getStringValue();
+            return parseNumbaVectorized(pyObject, numbaFuncTypes);
```

Comment:
I'm confused by this - are types[1] - types[n - 1] not used?

Reply:
added comments to clearly explain that we only use the first signature of the vectorized function